server/utils/placements/index.ts

```diff
@@ -211,5 +213,27 @@ export const renderKeyworkersSelectOptions = (
     })),
 ]
 
+export type PlacementTab = 'application' | 'assessment' | 'placementrequest' | 'placement' | 'timeline'
+
+export const tabTextMap: Record<PlacementTab, string> = {
+  application: 'Application',
+  assessment: 'Assessment',
+  placementrequest: 'Request for placement',
```

Comment:
CamelCase?

Reply:
I used lowercase as this literal goes into the querystring. Probably should be kebab-case but that's way too inconvenient.